dir-spec.txt

```diff
@@ -317,24 +317,28 @@
 
    The timeline for a given consensus is as follows:
 
-   VA-DistSeconds-VoteSeconds: The authorities exchange votes.
+   VA-DistSeconds-VoteSeconds: The authorities exchange votes. Each authority
+   uploads their vote to all other authorities.
 
    VA-DistSeconds-VoteSeconds/2: The authorities try to download any
-   votes they don't have. Furthermore, they stopped accepting vote posted to
-   them.
-
-      Note: The reason why the vote should be refused is to minimize the
-      chance of a consensus split if the authorities are under bandwidth
-      pressure. If an authority is struggling to upload its vote and finally
-      does it on a fraction of authorities after this period, they will
-      compute a consensus different from the others. By refusing the vote
-      after this period, we increase our chances that everyone will use the
-      same vote set.
-
-      It does not fix the problem entirely because the problem also exists if
-      N authorities are able to fetch a specific vote but M authorities fail
-      to do so. However, it is an improvement towards making sure each
-      authority has the same set of votes.
+   votes they don't have.
+
+   Authorities may also reject any votes that other authorities try to upload
```

Comment:
should "may" be "MAY"?  Should it be "SHOULD"?

Reply:
Changed to SHOULD in b4020cea54b3c6ce8ed8d7acfb13e1fc1d96b73d.